+dj/Relvar.m

```diff
@@ -64,22 +64,16 @@ function cleanup(self)
                 rels = [rels{:}];
                 rels(1) = rels(1) & self.restrictions;
                 
-                % apply proper restrictions
```

Comment:
What happened to these restrictions?

Reply:
this is absorbed by the restriction with `pro(rel(i))`

Reply:
This is inefficient and for now I must turned down this solution.

Reply:
Agreed - I would rather implement much more full fledged solution. Will go ahead and close this PR for now.